float32 pan
float32 tilt
---
bool success